jobserver/migrations/0006_move_to_new_release_flow.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.2.4 on 2023-08-31 16:13
+
+from django.db import migrations
+
+
+def move_workspaces_to_new_release_flow(apps, schema_editor):
+    Workspace = apps.get_model("jobserver", "Workspace")
+    Workspace.objects.all().update(uses_new_release_flow=True)
```

Comment:
Not worth changing for this but you don't need the `.all()` call here, it's a noop :)

Reply:
Good to know for the future!